libs/components/indicators/src/lib/modules/help-inline/help-inline.component.scss

```diff
@@ -15,20 +17,23 @@
   }
 }
 
-:host-context(.sky-control-help) {
-  .sky-help-inline {
-    padding: 0;
-  }
-}
-
 @include mixins.sky-theme-modern {
   .sky-help-inline {
+    border-radius: 3px;
     ::ng-deep .fa-stack-2x {
       font-size: 16px;
     }
 
     ::ng-deep .fa-stack-1x {
       font-size: 10px;
     }
+
+    @include modernMixins.sky-theme-modern-button-variant(
+      $sky-text-color-action-primary,
+      transparent,
+      transparent,
+      $sky-theme-modern-background-color-primary-dark,
+      0px 5px
```

Comment:
We want the padding to be 1px top/bottom and 5px left/right; however, the mixin adds 1px to the values passed in and so 0px and 5px are specified here.